ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -59,6 +60,7 @@
 )
 
 logger = logging.getLogger(__name__)
+SERVICE_CA_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
```

Comment:
Would it be an idea to move this to `settings`?

We could then set an environment variable, if needed, to _other_ paths (possibly in non-OpenShift environments).

It might be useful for the "Containerized install" work @mabashian @goneri have been doing.